hack/update/generate-crds.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+# Kindly borrowed from Kind
+# https://github.com/kubernetes-sigs/kind/blob/c5298b2/hack/update/generated.sh
+#
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# 'go generate's kind, using tools from vendor (go-bindata)
+set -o errexit -o nounset -o pipefail
+
+# cd to the repo root
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
+cd "${REPO_ROOT}"
+
+# build the generators using the tools module
+cd "hack/tools"
+"${REPO_ROOT}/hack/go_container.sh" go build -o /out/controller-gen sigs.k8s.io/controller-tools/cmd/controller-gen
+# go back to the root
+cd "${REPO_ROOT}"
+
+# turn off module mode before running the generators
+# https://github.com/kubernetes/code-generator/issues/69
+# we also need to populate vendor
+hack/go_container.sh go mod tidy
+hack/go_container.sh go mod vendor
+export GO111MODULE="off"
```

Comment:
Do we need this for Kubernetes > 1.16?

Reply:
No idea, and worth and investigation, but probably better to do this in a separate PR.